scipy/stats/tests/test_morestats.py

```diff
@@ -1471,7 +1471,7 @@ def test_wilcoxon_bad_arg(self):
         assert_raises(ValueError, stats.wilcoxon, [1, 2], [1, 2], "dummy")
         assert_raises(ValueError, stats.wilcoxon, [1, 2], [1, 2],
                       alternative="dummy")
-        assert_raises(ValueError, stats.wilcoxon, [1]*10, mode="xyz")
+        assert_raises(ValueError, stats.wilcoxon, [1]*10, method="xyz")
```

Comment:
`method` is the new name for `mode`. I didn't change the tests when the change was made, but now seems like a good time to do it. In `test_approx_mode` below, I test that `mode` is still accepted.